cmd/argo/commands/get.go

```diff
@@ -88,6 +88,7 @@ func NewGetCommand() *cobra.Command {
 
 	command.Flags().StringVarP(&getArgs.output, "output", "o", "", "Output format. One of: json|yaml|wide")
 	command.Flags().BoolVar(&noColor, "no-color", false, "Disable colorized output")
```

Comment:
where else is `--no-color` an option?

Reply:
There is also a `--no-color` option on the `argo logs` command but I don’t think this command generates any utf-8 character.